openquake/hazardlib/geo/surface/simple_fault.py

```diff
@@ -218,19 +214,14 @@ def get_fault_patch_vertices(cls, fault_trace, upper_seismogenic_depth,
             all_lons = numpy.array(lons + list(reversed(t_lon)), float)
             all_lats = numpy.array(lats + list(reversed(t_lat)), float)
             all_deps = numpy.array(deps + list(reversed(t_dep)), float)
-
-        p0 = Point(all_lons[index_patch - 1], all_lats[index_patch - 1],
-                   all_deps[index_patch - 1])
-        p1 = Point(all_lons[index_patch], all_lats[index_patch],
-                   all_deps[index_patch])
-        p2 = Point(all_lons[2 * len(fault_trace) - (index_patch + 1)],
-                   all_lats[2 *
-                   len(fault_trace) - (index_patch + 1)],
-                   all_deps[2 * len(fault_trace) - (index_patch + 1)])
-        p3 = Point(all_lons[2 * len(fault_trace) - index_patch],
-                   all_lats[2 * len(fault_trace) - index_patch],
-                   all_deps[2 * len(fault_trace) - index_patch])
-
+        index1 = int(index_patch - 1)
+        index2 = int(index_patch)
+        index3 = int(2 * len(fault_trace) - (index_patch + 1))
+        index4 = int(2 * len(fault_trace) - index_patch)
```

Comment:
this is to make sure the index is in integer format to avoid the possible causing error in different numpy version.

I will modify the changelog after the LGTM to avoid the possible file conflict while merging.